recipe/bld.bat

```diff
@@ -27,6 +26,11 @@ IF "%build_with_cuda%" == "" goto cuda_end
 set "PATH=%CUDA_BIN_PATH%;%PATH%"
 set CUDNN_INCLUDE_DIR=%LIBRARY_PREFIX%\include
 
+for /f "tokens=* usebackq" %%f in (`where nvcc`) do (set "dummy=%%f" && call set "NVCC=%%dummy:\=\\%%")
+set "NVCC=%NVCC% --use-local-env"
+set CMAKE_CUDA_FLAGS="%CMAKE_CUDA_FLAGS% --use-local-env "
```

Comment:
The project is overriding this variable [upstream](https://github.com/lightvector/KataGo/blob/42bd404d4124afeb8c9bb2422e0354927c81c032/cpp/CMakeLists.txt#L51) :( You might need to patch.

Reply:
Thanks for spotting this. I pushed a patch.